lib/relay.js

```diff
@@ -55,8 +55,33 @@ function requestHandler(filterRules) {
 function responseHandler(filterRules, config) {
   const filters = Filters(filterRules);
 
+  const supportedFiles = (filterRules || [])
+    .filter(f => f.path && f.path.includes('/contents/'))
+    .map(f => f.path.split('/').pop());
+
   return (brokerToken) => ({ url, headers, method, body = null } = {}, emit) => {
-    // run the request through the filter
+
+    // Changing old /raw endpoint to /contents api for GH clients.
+    // Since GH enterprise 2.11 version the /raw endpoint is deprecated
+    // This piece of code allow us to move to the new /content API without
+    // breaking old broker clients.
+    // Once all our GHE client will move to this new version of the broker
+    // client we could remove this code.
+    function rewriteRawApi() {
+      // find urls that ends with manifest files
+      const file = supportedFiles.find(f => url.endsWith(f));
+      if (config.GITHUB && file && url) {
+        headers.accept = 'application/vnd.github.2.11.raw';
+        // switching urls from: /:name/:repo/:branch*/:path
+        // to /repos/:name/:repo/contents/:path?ref=:branch
+        // $1 - repo/owner, $2 - branch, $3 - path
+        url = url
+          .replace(/(\/.+?\/.+?\/)(.+?)(\/.*)/, `/repos$1contents$3?ref=$2`);
+      }
+    }
+
+    rewriteRawApi();
```

Comment:
Call `rewriteRawApi` only if `config.GITHUB` - this way we skip `supportedFiles.find` on every request for non-GH brokers. Remove `config.GITHUB` from line 73.